src/main/resources/hudson/plugins/git/UserMergeOptions/config.jelly

```diff
@@ -13,4 +13,7 @@
   <f:entry title="Merge strategy:" field="mergeStrategy">
       <f:select />
   </f:entry>
-</j:jelly>
\ No newline at end of file
+  <f:entry title="Fast Forward Mode:" field="fastForwardMode">
+    <f:select />
```

Comment:
Have you tried https://jenkins-ci.org/maven-site/jenkins-core/jelly-taglib-ref.html#form:enum ?